src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -975,6 +975,21 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             .context("`image` is not ORG/NAME:VERSION")?
             .to_string();
 
+        let container_security_context = Some(SecurityContext {
+            privileged: Some(false),
+            run_as_non_root: Some(true),
+            allow_privilege_escalation: Some(false),
+            seccomp_profile: Some(SeccompProfile {
+                type_: "RuntimeDefault".to_string(),
+                ..Default::default()
+            }),
+            capabilities: Some(Capabilities {
+                drop: Some(vec!["ALL".to_string()]),
```

Comment:
Do any of these fields need to be configurable by the cloud layer? E.g., via CLI flags?

Reply:
I'm of two minds on this - 

Generally, I don't think it makes sense for these to be configurable, since to meet the `restricted` profile requirements they need to be all set as they are. 

I did (and am) considering whether there should be a flag like  `enable-restricted-pod-standard` that would enable and disable the entire block, but it slightly feels like a footgun since if it's disabled in MZ but enabled in cloud MZ won't be able to deploy, whereas there is no downside to having PSS enabled in MZ but disabled in cloud.

Reply:
> whereas there is no downside to having PSS enabled in MZ but disabled in cloud.

Hm I think it's a bit more than that -- all of these security context changes will still have an effect even without pod security standards enforcement. It might be nice to have them configurable through some mechanism if we need to roll back the change, separate from whether cloud is enforcing a policy on what those security context changes ought to look like.

Reply:
> if we need to roll back the change

Yeah, I addressed that above. I think there should be a way to disable all of the PSS configuration, but probably not cli/env/ld flags for each one. 

Because if for example, 9 environments work perfectly, and 1 fails because of the PSS, we won't be able to loosen the security for a single environment once the cloud side is enforcing these, so we'd have to disable it for everyone anyway.